tests/ratio_estimator_test.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import pytest
+import torch
+from torch import eye, zeros
+from torch.distributions import MultivariateNormal
+
+from sbi.neural_nets.classifier import build_linear_classifier
+from sbi.neural_nets.ratio_estimators import TensorRatioEstimator
+
+
+class EmbeddingNet(torch.nn.Module):
+    def __init__(self, shape: torch.Size) -> None:
+        super().__init__()
+        self.shape = shape
+
+    def forward(self, x):
+        for _ in range(len(self.shape) - 1):
+            x = torch.sum(x, dim=-1)
+        return x
+
+
+@pytest.mark.parametrize("ratio_estimator", (TensorRatioEstimator,))
+@pytest.mark.parametrize(
+    "theta_shape", ((1,), (2,), (1, 1), (2, 2), (1, 1, 1), (2, 2, 2))
+)
+@pytest.mark.parametrize("x_shape", ((1,), (2,), (1, 1), (2, 2), (1, 1, 1), (2, 2, 2)))
+def test_api_ratio_estimator(ratio_estimator, theta_shape, x_shape):
```

Comment:
Please add a test that checks if 2D data (images) can be handled with [appropriate embedding nets](https://github.com/sbi-dev/sbi/blob/0f0ee6e9f6c35a6d030281427409bcfacc0e4764/sbi/neural_nets/embedding_nets.py#L106).